api4/integrity_local.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
I think we should move the handler to `system_local.go` so it's placed in the same file as the endpoint itself

Reply:
Absolutely, I think I did back and forth and was not making up my mind. 
Thanks for pointing it out, will change :+1: